udata/auth/__init__.py

```diff
@@ -39,18 +35,10 @@ def __init__(self, *needs):
 admin_permission = Permission()
 
 
-@bp.before_app_request
-def ensure_https_authenticated_users():
-    # Force authenticated users to use https
-    if (not current_app.config.get('TESTING', False)
-            and current_app.config.get('USE_SSL', False)
-            and current_user.is_authenticated()
-            and not request.is_secure):
-        return redirect(request.url.replace('http://', 'https://'))
-
-
 def init_app(app):
+    from .views import auth
```

Comment:
Maybe we should add a comment every time we cross a circular import from now on in order to identify which files are critical and try to split those in the future, @noirbizarre what do you think?